server/middleware/searchHandler.js

```diff
@@ -47,10 +48,7 @@ async function handleSearch(req, res, next) {
             .sort(sortObjectByProp(workstackCase => workstackCase.caseReference))
             .map(bindDisplayElements(fromStaticList)));
         const { workstackTypeColumns } = await req.listService.fetch('S_SYSTEM_CONFIGURATION');
-        // using DEFAULT columns for search
-        const workstackTypeForSearch = workstackTypeColumns.find(
-            item => item.workstackType === 'DEFAULT'
-        );
+        let workstackTypeForSearch = returnSearchWorkstackColumns(workstackTypeColumns, request);
```

Comment:
Can this be `const`?
```suggestion
        const workstackTypeForSearch = returnSearchWorkstackColumns(workstackTypeColumns, request);
```

Might be nicer to call it getSearchWorkstackColumns?

Reply:
I have addressed this